src/models/contracts.js

```diff
@@ -2,10 +2,12 @@
 
 const mongoose = require('mongoose');
 
-const Schema = mongoose.Schema;
+const Schema = mongoose.Schema,
+      ObjectId = Schema.ObjectId;
+
 const ContractsSchema = new Schema({
   _id: {
-    type: ObjectId,
+    type: Number,
```

Comment:
This was giving me "ObjectId" is not defined (I added it above), but also: 

https://stackoverflow.com/questions/8111846/how-to-set-objectid-as-a-data-type-in-mongoose

Reply:
I double checked and the _id is not necessary. Provide it is not a problem, but if we dont specify it, a field _id with type ObjectId get created in the new document saved in the DB.